pkg/wireguard/wireguard.go

```diff
@@ -77,7 +69,7 @@ func (dev *Device) Run(ctx context.Context, logger *zap.Logger, peers PeerSource
 		Errorf:   logger.Sugar().Errorf,
 	}
 
-	uapi, err := ipc.UAPIListen(interfaceName, dev.fileUAPI)
+	uapi, err := UAPIOpen(interfaceName)
```

Comment:
so go automatically picks the right method definition based on os?

Reply:
yep, it build either one of the files wireguard_*.go based on the `GOOS`

Reply:
oh nvm, just saw the build tags